modules/core/src/main/scala/org/scalasteward/core/update/UpdateService.scala

```diff
@@ -181,4 +181,20 @@ object UpdateService {
       case ("org.xerial.sbt", "sbt-pack")                    => false
       case _                                                 => true
     }
+
+  // (OldGroupId, ArtifactId) -> NewGroupId
+  final private val mapToNewGroupId = Map(
+    ("org.spire-math", "kind-projector") -> "org.typelevel",
```

Comment:
I'd replace the second field in the key of this map with a predicate. In this case, `== "kind-projector"`, but if we were to add zio, we could do pretty much `("org.scalaz.zio", _ => true) -> "dev.zio"`.

Or just replace this with a partial function altogether and use a pattern match :)

Reply:
Thanks !!
It helped me to understand that there could be needs for using new organization for multiple artifacts.

Replaced with pattern matching in https://github.com/fthomas/scala-steward/pull/676/commits/21fa1687b453401e53cf41c1d7e0dfbbaad77403 so one can add pattern easily like
```scala
case ("org.scalaz.zio", _) => "dev.zio"
case ("com.example", artifactId) if Set("foo", "bar")(artifactId) => "com.example.new"
```

Reply:
Oops, I played myself. Zio was never part of the organization before. But a pattern match for modules starting with a specific prefix is still a thing!